.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+Thanks for submitting your Pull Request!
+
+Please make sure your PR meets the following requirements:
+
+- [ ] Pull Request title is properly formatted: `[WFLY-XYA] Subject`
+- [ ] Pull Request contains link to the JIRA issue
+- [ ] Pull Request contains description of the issue
+- [ ] Pull Request does not include fixes for issues other than the main ticket
```

Comment:
I think moving to PRs representing complete features I think it is going to be unrealistic to expect PRs never contain related fixes.

There are approaches that can be used in git to submit those fixes independently as well but they may be needed in the topic branch being submitted for that branch to build and pass the testsuite.

Reply:
You can always keep this item not selected and explain more in PR description. You can think about this as questionnaire with input for RC / gate-keeper for easier checks

I believe we should not mix too many thing together, it make back-porting and git bisect complicated